src/screens/KeyManagement/OCR2Keys.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { gql, useMutation } from '@apollo/client'
+import { useDispatch } from 'react-redux'
+
+import { deleteSuccessNotification } from './notifications'
+import { OCR2KeysCard } from './OCR2KeysCard'
+import { useOCR2KeysQuery } from 'src/hooks/queries/useOCR2KeysQuery'
+import { useQueryErrorHandler } from 'hooks/useQueryErrorHandler'
+
+export const DELETE_OCR2_KEY_BUNDLE_MUTATION = gql`
+  mutation DeleteOCR2KeyBundle($id: ID!) {
+    deleteOCR2KeyBundle(id: $id) {
+      ... on DeleteOCR2KeyBundleSuccess {
+        bundle {
+          id
+        }
+      }
+    }
+  }
+`
+
+/**
+ * This follows the form and structure of OCRKeys but
+ */
+export const OCR2Keys = () => {
+  const dispatch = useDispatch()
+  const { handleQueryError } = useQueryErrorHandler()
+  const { data, loading, error, refetch } = useOCR2KeysQuery({
+    fetchPolicy: 'network-only',
+    onError: handleQueryError,
+  })
+
+  const [deleteOCR2KeyBundle] = useMutation<
+    DeleteOcr2KeyBundle,
+    DeleteOcr2KeyBundleVariables
+  >(DELETE_OCR2_KEY_BUNDLE_MUTATION)
+
+  const handleDelete = async (id: string) => {
+    try {
+      const result = await deleteOCR2KeyBundle({ variables: { id } })
+
+      const payload = result.data?.deleteOCR2KeyBundle
+      switch (payload?.__typename) {
+        case 'DeleteOCR2KeyBundleSuccess':
+          dispatch(
+            deleteSuccessNotification({
+              keyType: 'Off-ChainReporting Key Bundle',
+            }),
+          )
+
+          refetch()
+
+          break
+      }
+    } catch (e) {
+      handleQueryError(e)
+    }
+  }
+
+  return (
+    <OCR2KeysCard
+      loading={loading}
+      data={data}
+      errorMsg={error?.message}
```

Comment:
Do we still need this prop if we have an error handler already set?

Reply:
Removed in [`2079e2a` (#26)](https://github.com/smartcontractkit/operator-ui/pull/26/commits/2079e2a0f719b07775cdfe89764f91e91e9ea3aa)